docs/development_suite/set-up-infrastructure/enhanced-project-workflow.md

```diff
@@ -87,6 +81,14 @@ Versions can only be deleted by Project Administrators.
 
 ![Versions management page](img/versions-overview.png)
 
+### Revision history
+
+To keep track of the Project evolution, the user can create `versions` (similar to Git tags) that act as immutable snapshots of the configuration in a particular moment. To keep track of how a revision has changed over time, the user can access its [revision history](#revision-history), which contains past snapshots of the Project configuration.
```

Comment:
I don't think it is necessary to link the #revision-history section here, since we are already inside it

Reply:
Thanks for the catch. Probably a mistake after the merge from main.